src/vl-tabs.js

```diff
@@ -187,6 +203,7 @@ export class VlTabs extends vlElement(HTMLElement) {
     const tabPanesToAdd = mutations.flatMap((mutation) => [...mutation.addedNodes]).filter((node) => node instanceof VlTabsPane);
     tabPanesToAdd.forEach((tabPane) => {
       const index = this.__tabPanes.indexOf(tabPane);
+      this._moveTabPaneTitleSlot(tabPane);
       this._addTab({id: tabPane.id, title: tabPane.title, index: index});
```

Comment:
Denk duplicatie hier van het feit dat je altijd moveTabPaneTitleSlot moet doen als je addTab doet?

Reply:
deduplicatie _moveTabPaneTitleSlot methode in vl-tabs. Deze wordt nu enkel opgeroepen in de _addTab methode